sia/upload_test.go

```diff
@@ -39,4 +56,10 @@ func testUploadFile(t *testing.T, c *Core) {
 	if c.hostDB.Size() < 1 {
 		t.Fatal("Hostdb got pruned while trying to make a contract?")
 	}
+
+	// Check that the file can be downloaded.
+	err = c.renter.Download("one", "renterDownload")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
I don't want another `filesCreatedDuringTesting`. I guess it doesn't hurt anything to keep it in the gitignore, but I'd still like to see it removed.